backend/django/core/utils/util.py

```diff
@@ -103,11 +110,14 @@ def upload_data(form_data, project, queue=None, irr_queue=None, batch_size=30):
             irr_percent=project.percentage_irr,
             batch_size=batch_size,
         )
-    transaction.on_commit(
-        lambda: tasks.send_label_embeddings_task.apply_async(
-            kwargs={"project_pk": project.pk}
-        )
-    )
+
+    # Celery and emebeddings wasn't working well together...
```

Comment:
@dsteedRTI for cleaner code let's drop the commented out stuff and add more details to the comments themselves